distribution/docs/pom.xml

```diff
@@ -119,6 +122,42 @@
     </profiles>
     <build>
         <plugins>
+            <plugin>
+                <groupId>org.codehaus.gmaven</groupId>
+                <artifactId>groovy-maven-plugin</artifactId>
+                <version>2.1</version>
+                <inherited>false</inherited>
+                <executions>
+                    <execution>
+                        <id>aggregate-deps</id>
+                        <phase>generate-sources</phase>
+                        <goals>
+                            <goal>execute</goal>
+                        </goals>
+                        <configuration>
+                            <source>
+                                def lines = []
+                                new File('${highest-basedir}').traverse(type: groovy.io.FileType.FILES, nameFilter: ~/maven-deps.txt/) { file ->
+                                    for (line in file.readLines())
+                                        lines.add(line.trim())
+                                }
+                                def output = ""
+                                Outer:
+                                for (line in lines.unique().sort()) {
+                                    if (!(line in ${dependency-ignored-lines})) {
+                                        for (ignoredPackage in ${dependency-ignored-packages}) {
+                                            if (line.startsWith(ignoredPackage))
+                                                continue Outer
+                                        }
+                                        output += "* " + line + System.getProperty("line.separator")
```

Comment:
In groovy if you are using a GString (Double quotes) instead of a java.lang.String (single quotes) then you can use String Interpolation.  EG
```groovy
output += "* ${line}${System.lineSeparator()}"
```